doc/INSTALL.md

```diff
@@ -212,6 +212,29 @@ Howto create a release (core-contributor rights needed)
 10. *check release* : an automatic workflow started on github and create the new release of plugin
 11. `docker-compose.yml` : check and modify (if needed) the version in this file to the new SNAPSHOT version
 
+Howto publish new release on SonarQube Marketplace
+--------------------------------------------------
+
+1. Create a fork of [SonarSource/sonar-update-center-properties](https://github.com/SonarSource/sonar-update-center-properties.git)
+2. Change corresponding plugin metadata file (for `ecocode-java`: [ecocodejava.properties](https://github.com/SonarSource/sonar-update-center-properties/blob/master/ecocodejava.properties), for `ecocode-php`: [ecocodephp.properties](https://github.com/SonarSource/sonar-update-center-properties/blob/master/ecocodephp.properties), for `ecocode-python`: [ecocodepython.properties](https://github.com/SonarSource/sonar-update-center-properties/blob/master/ecocodepython.properties)): 
+   * Append new version to `publicVersions` value (comma separated value)
```

Comment:
Great idea! Do we also need to add the old version to the key `archivedVersions`?

Reply:
@utarwyn @jycr 
I've just approved the PR, you can merge it.
Maybe add an answer to @utarwyn question before merging it ?